bin/build-image

```diff
@@ -2,6 +2,10 @@
 
 set -e
 
+# Remove cached bytecode Python files, to ensure latest code is used
+find . | grep -E "(__pycache__|\.pyc|\.pyo$)" | xargs rm -rf
```

Comment:
Not necessarily something for this PR, but I had a thought this morning - we could probably update the `.dockerignore` file in this repository to also not copy these files into the image being built.

Reply:
that's a good idea.